### DISCUSSION TITLE
Cables for client work

### DISCUSSION THREAD

**Discussion by positlabs** [cables-gl/cables_docs#194] [Answers & Questions] [answered]
Can we use cables for client work? What are the terms of service and licenses for cables?

**Comment by pandrr** [comment, marked as answer]
cables is free to use, feel free to use it for client work

see: https://cables.gl/docs/faq/licence_payment/licence_payment

you can always talk and book us (https://undev.studio) to develop new features etc.

**Comment by positlabs** [comment]
Niiiice! Good to know :)
